File: superstitioApi/java/superstitioapi/powers/interfaces/invisible/InvisiblePower_InvisibleTips.java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by FernFlower decompiler)
//

package superstitioapi.powers.interfaces.invisible;

import com.megacrit.cardcrawl.powers.AbstractPower;

public interface InvisiblePower_InvisibleTips {
    static boolean shouldInvisibleTips(AbstractPower power) {
        if (power instanceof InvisiblePower_InvisibleTips)
            return ((InvisiblePower_InvisibleTips) power).checkShouldInvisibleTips();
        return false;
    }

    default boolean checkShouldInvisibleTips() {
        return true;
    }
}

File: superstitioApi/java/superstitioapi/powers/interfaces/invisible/InvisiblePower_InvisibleTips.kt
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by FernFlower decompiler)
//
package superstitioapi.powers.interfaces.invisible

import com.megacrit.cardcrawl.powers.AbstractPower

interface InvisiblePower_InvisibleTips
{
    fun checkShouldInvisibleTips(): Boolean
    {
        return true
    }

    companion object
    {
        fun shouldInvisibleTips(power: AbstractPower): Boolean
        {
            if (power is InvisiblePower_InvisibleTips)
                return power.checkShouldInvisibleTips()
            return false
        }
    }
}

